### PR TITLE
AAP-32327: adds redis config info to RPM installer guide

### PR DESCRIPTION
**Pull request by hherbly**
See [AAP-32327](https://issues.redhat.com/browse/AAP-32327) for more info.

### CODE REVIEW

File: downstream/modules/platform/ref-general-inventory-variables.adoc [hunk 1]
```diff
@@ -26,6 +26,9 @@ Default = `TLSv1.2`
 Each element in the list is provided into `http nginx config` as a separate line. 
 
 Default = empty list
+| `redis_mode` | | Redis can be colocated with any other node in a clustered installation.
```
Comment (jamesmarshall24) [inline downstream/modules/platform/ref-general-inventory-variables.adoc:29]: Redis cannot be colocated with automation controller, automation controller execution, or database nodes

Comment (jamesmarshall24) [reply downstream/modules/platform/ref-general-inventory-variables.adoc:29]: They can only be automationgateway, automationhub, automationedacontroller nodes

Comment (hherbly) [reply downstream/modules/platform/ref-general-inventory-variables.adoc]: Gotcha. Updated

**Review by jamesmarshall24 — APPROVED** [review]

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**